packages/apollo/src/commands/client/codegen.ts

```diff
@@ -50,7 +50,7 @@ export default class Generate extends ClientCommand {
     }),
     addTypename: flags.boolean({
       description: "Automatically add __typename to your queries",
-      default: true
+      default: false
```

Comment:
Let's keep this as `true` but add `allowNo: true` and update the description to tell users they can use `--no-addTypename` to disable this feature

Reply:
Ok, I agree. Didn't know that `true` is default for Apollo Client. Will update the description also, because documentation was a little off, especially for this part.

Should i create new PR only for this change, and close this one?